src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1212,6 +1212,28 @@ private bool RequiresReparse(ScriptParseInfo info, ScriptFile scriptFile)
                 || !string.Equals(prevSqlText, currentSqlText);
         }
 
+        private bool RefreshNeeded(string uri)
+        {
+            ConnectionInfo connInfo;
+            System.Data.Common.DbConnection connection;
+            connectionService.TryFindConnection(uri, out connInfo);
```

Comment:
we do support intellisense for disconnected scenario, your change will break it, you should handle the scenario when the return value is false (disconnected scenario) and return false.

Reply:
this comment was resolved without code changes or follow up comments. reactivating it.

Reply:
I changed it to false

Reply:
I will update it because that doesn't seem right either